gossipd/gossip.c

```diff
@@ -1627,6 +1629,32 @@ static struct io_plan *conn_init(struct io_conn *conn, struct reaching *reach)
 	return io_connect(conn, &ai, connection_out, reach);
 }
 
+static struct addrhint *
+seed_resolve_addr(const tal_t *ctx, const struct pubkey *id, const u16 port)
+{
+	struct addrhint *a;
+	char bech32[100], *addr;
+	u8 der[PUBKEY_DER_LEN];
+	u5 *data = tal_arr(ctx, u5, 0);
+
+	pubkey_to_der(der, id);
+	bech32_push_bits(&data, der, PUBKEY_DER_LEN*8);
+	bech32_encode(bech32, "ln", data, tal_count(data), sizeof(bech32));
+	addr = tal_fmt(ctx, "%s.lseed.bitcoinstats.com", bech32);
```

Comment:
Probably better to put `lseed.bitcoinstats.com` in an option, but maybe OK as first cut, can always do later.